spec.bs

```diff
@@ -2049,6 +2054,24 @@ If a specification has an Infra value (e.g., after using [=parse a JSON string t
 
 Specifications may wish to leave room in their formats to accept options for {{URLPatternOptions}}, override the base URL, or similar, since it is not possible to construct a {{URLPattern}} object directly in this case, unlike in a JavaScript API. For example, <cite>Speculation Rules</cite> accepts a "`relative_to`" key which can be used to switch to using the [=document base URL=] instead of the JSON resource's URL. [[SPECULATION-RULES]]
 
+<h3 id=other-specs-http>Integrating with HTTP header fields</h3>
+
+HTTP headers which include URL patterns should accept a string in the constructor string syntax, likely as part of a structured field [[RFC8941]].
```

Comment:
What is "constructor string syntax"? Is this talking about the `URLPattern` constructor which it below says should not be relied upon?

Reply:
It refers to the syntax that is accepted when a single string is passed to the constructor (i.e., is accepted by the [constructor string parser](https://urlpattern.spec.whatwg.org/#constructor-string-parsing)), as opposed to the pattern syntax for a single component.

It could be renamed something else, but this is how the spec currently refers to that syntax.

Reply:
I think if the syntax is defined somewhere and you can xref it that seems reasonable. Otherwise it seems rather opaque to me.

Reply:
Added links to the constructor string parsing section.